README.md

```diff
@@ -26,6 +26,8 @@ To build it, run `go build`. To test, run `go test ./...`.
 
 > Requires golang 1.12+.
 
+Additional unit tests are in the [test](test) directory. First built with `go build`, then run them using [BATS](https://github.com/bats-core/bats-core) with `PATH=.:$PATH bats test`.
```

Comment:
This is a pretty good idea. Only concern is that the stuff in `test` tests the Wash binary, so they're not really unit tests. They seem more like acceptance tests?

Reply:
Ah yes, wrote the wrong thing.